### PR TITLE
Use DI in views/controllers

### PR DESCRIPTION
**Pull request by palequezal**

### CODE REVIEW

File: app/views/agriculturalFlatRateScheme.scala.html [hunk 1]
```diff
@@ -37,7 +44,7 @@
         @link(appConfig.VATNotice700_46agriculturalURL, "agriculturalFlatRateScheme.linkText", isExternal = true)
     }
 
-    @helpers.form(action = AgriculturalFlatRateSchemeController.onSubmit(), 'autoComplete -> "off") {
+    @formWithCsrf(action = AgriculturalFlatRateSchemeController.onSubmit(), 'autocomplete -> "off") {
```
Comment (jawilliams23) [inline app/views/agriculturalFlatRateScheme.scala.html:47]: Should "autocomplete" be "autoComplete" here? It is in other files

**Review by jawilliams23 — COMMENTED** [review]
Should the "autocomplete" on the agriculturalFlateRateScheme view be "autoComplete"? It's in the latter format for every other pages

File: app/views/agriculturalFlatRateScheme.scala.html (hunk 1 above)
Comment (palequezal) [reply app/views/agriculturalFlatRateScheme.scala.html:47]: Good spot! I deleted that line by mistake and re-added it with a bonus typo.

Comment (jawilliams23) [reply app/views/agriculturalFlatRateScheme.scala.html:47]: Gotcha! Once that's re-pushed we're all good to merge.

**Pull request merged**